tests/shared/src/test/scala/scala/meta/tests/contrib/ScaladocParserSuite.scala

```diff
@@ -236,4 +236,19 @@ class ScaladocParserSuite extends FunSuite {
     assertEquals(result, expectation)
   }
 
+  test("multiline param") {
+    val description = "This description starts on the next line."
+    val result = parseString(
+      s"""|/** @param a
+            *    $description
+            */
+         |case class foo(a: String)
+         |""".stripMargin
```

Comment:
please run `./bin/scalafmt` on each commit and resubmit

Reply:
Done.